src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1408,29 +1428,29 @@ public override void _Notification(int what)
         }
     }
 
-    public void ShowOrganelleMenu(OrganelleTemplate selectedOrganelle)
+    public void ShowOrganelleMenu(OrganelleTemplate main, List<OrganelleTemplate?> selectedOrganelles)
```

Comment:
Removing nulls is good, but this parameter cannot be an `IEnuerable` because this method modifies the list by moving the main organelle to the head of the list.

Reply:
That's very unexpected and unintuitive that the given list would be modified... isn't there some better design that can be used here?

Reply:
I can move the responsibillity to the caller, see https://github.com/Revolutionary-Games/Thrive/pull/2445/commits/bd5378862246a940d72482fc905e0a4243bad39f, do you think this is better?

Reply:
I think it's better. If there will be mutliple places this might be called from it might be better to return the new list from this method so that the caller can then use the return value and put it where it is needed.